src/output/mod.rs

```diff
@@ -36,6 +38,12 @@ pub enum Mode {
     Lines,
 }
 
+#[derive(PartialEq, Eq, Debug)]
+pub enum OutputType {
+    Legacy,
+    Json,
+}
```

Comment:
I wouldn't call the default output type `Legacy`. This should be something like Console or TTY?